### DISCUSSION TITLE
Zwavejs2MQTT LXC

### DISCUSSION THREAD

**Discussion by kulekimba** [tteck/Proxmox#220] [Q&A] [answered]
Hello

Running the Zwavejs2MQTT LXC and trying to run the update. 
When i do so, nothing happen and it kinda freezes with message: 
Updating Zwavejs2MQTT... 
Nothing more i happend, and i do need to reboot the Container

Any tip to get around this?

**Comment by tteck** [comment]
Just checked...
```
root@zwavejs2mqtt:~# bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/zwavejs2mqtt-update.sh)"
 Updating Zwavejs2MQTT... ✔ 
 Checking Service... ✔ 
 Cleanup... ✔ 

 Finished 
root@zwavejs2mqtt:~# 
```
```
zwavejs2mqtt: 6.9.0
zwave-js: 9.2.1
home id: undefined
home hex: undefined
```

**Comment by iam-DM** [comment]
I have the same problem when trying to update.

I can see that zwavejs2mqtt-v6.9.0.zip has been downloaded. But it does not continue with the installation

**Comment by tteck** [comment]
Maybe unzip isn't installed? Run `apt install unzip` in the zwavejs2mqtt LXC console. Then re-run the update

**Reply by iam-DM** [reply]
Unfortunately, that did not solve the issue.

Since I have no experience with linux, I have no idea how to troubleshoot

**Comment by tteck** [comment, marked as answer]
Update script fixed
